html-css/lesson1.md

```diff
@@ -151,14 +154,11 @@ You can also set a transparent border, so that it takes up the space without sho
 
 ## Homework
 
-1. (Est. 2-3 hours) Create a new webpage (`.html`) on a subject that you're interested in. The page should not use any framework like Bootstrap and it should include the following elements:
-  - A linked `.css` file applies a consistent color scheme to links and text on your page.
-  - A page header with a title and description for the site.
-  - An articles section with three articles, each including a title, summary and a link.
-  - Proper use of the semantic HTML tags we discussed.
-  - Research the `:first-child` pseudo-class and use it to style the first article so that it stands out from the others.
+1. (Est. 2 hours) Read about [advanced CSS selectors](http://learn.shayhowe.com/advanced-html-css/complex-selectors/) and then practice by playing this [CSS selector game](https://flukeout.github.io/). It gets hard at the end, but try your best!
+
+2. Fork this repo and follow the instructions on the README - https://github.com/Code-Your-Future/html-css-project
 
-2. **Optional** (Est. 1 hour) Read about [advanced CSS selectors](http://learn.shayhowe.com/advanced-html-css/complex-selectors/) and then practice by playing this [CSS selector game](https://flukeout.github.io/). It gets hard at the end, but try your best!
+(If you need help with Forking - then refer back to [this tutorial](https://help.github.com/articles/fork-a-repo/))
```

Comment:
I was thinking, maybe we can add a third one add a third one with an **optional** tag? This will introduce some of them to flexbox, which will be covered in the next lesson:

```
Try changing the appearance of the `bikes-for-refugees` page without touching the HTML. Can you move the sidebar to the left side of the page? Can you reverse the order of "upcoming events", by putting "help us cycle" first and "join us for our fundraiser" last?
```